src/ui-config.ts

```diff
@@ -0,0 +1,5 @@
+export const configPromise = fetch('/pulp-ui-config.json').then((data) =>
+  data.json(),
+);
```

Comment:
Should we add an error handle for 404? Else the error message on 404 will be JSON parse error.